"Could not find com.android.support:support-annotations:21.0.0."

I have everything set up correctly -- SDK, NDK, local.properties, all that -- but I think the file gradle is needing is missing suddenly? I just get this error:

    $ ./tools/build-debug --info
    
    WARNING [Project: :libtermexec] Current NDK support is deprecated.  Alternative will be provided in the future.

    FAILURE: Build failed with an exception.

    * What went wrong:
    A problem occurred configuring project ':libtermexec'.
    > Could not resolve all dependencies for configuration ':libtermexec:_debugCompile'.
       > Could not find com.android.support:support-annotations:21.0.0.
         Searched in the following locations:
             https://jcenter.bintray.com/com/android/support/support-annotations/21.0.0/support-annotations-21.0.0.pom
             https://jcenter.bintray.com/com/android/support/support-annotations/21.0.0/support-annotations-21.0.0.jar
         Required by:
             Android-Terminal-Emulator:libtermexec:unspecified
    
    * Try:
    Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
    
    BUILD FAILED

--- a/tools/install-sdk-packages
+++ b/tools/install-sdk-packages
@@ -20,4 +20,4 @@ ANDROID="$ANDROID_SDK_ROOT/tools/android"
 # The update sdk command takes a comma-separated list of packages to install.
 # Installing them all at once may simplify the process of agreeing to licenses.
 
-$ANDROID update sdk -u -a -t android-11
+$ANDROID update sdk -u -a -t android-11,extra-android-m2repository

